pkgs/development/python-modules/humanize/default.nix

```diff
@@ -22,7 +22,6 @@ buildPythonPackage rec {
     homepage = https://github.com/jmoiron/humanize;
     license = licenses.mit;
     maintainers = with maintainers; [ ];
-    platforms = platforms.linux; # can only test on linux
```

Comment:
I think you want to put `platforms.unix` here, not remove the `platforms` specification.

Reply:
Good catch, will fix.

Reply:
I think it's safe to remove it since `humanize` only depends on python, and by default [`buildPythonPackage` uses python.meta.platforms](https://github.com/nixos/nixpkgs/blob/797e0d84a180dd24b7572ffef9a52677860970cb/pkgs/development/interpreters/python/mk-python-derivation.nix#L106)